landlab/components/bedrock_landslider/bedrock_landslider.py

```diff
@@ -342,6 +342,21 @@ def __init__(
             np.random.seed(seed)
 
     # Getters for properties
+    @property
+    def fraction_fines_LS(self):
```

Comment:
@loroberge What do you think about renaming this property to just `fraction_fines`? Names should be all lowercase and I'm not sure what *LS* stands for. If it stands for landslider, then that's probably redundant since it's an attribute of the landslider component.

Reply:
That sounds good. I believe this name was to differentiate it from a similar parameter used in `Space` and `SpaceLargeScaleEroder` (it is called `F_f` in those). You're right that it seems redundant.
@BCampforts do you have a preference for the property name?

Reply:
I'm now looking at PR #1649  with similar requests and I may have misunderstood. Do you want this only done for the public property? Here I've changed all instances of `fraction_fines_LS`....

Reply:
I was just thinking of the public methods.

Reply:
ok. I've fixed it to only change the public method.